packages/graphql-hooks/test/unit/GraphQLClient.test.ts

```diff
@@ -9,9 +9,6 @@ import {
   createMockSubscriptionClient
 } from '../utils'
 
-// workaround for https://github.com/octet-stream/form-data/issues/50
-const { fileFromPathSync } = require('formdata-node/lib/cjs/fileFromPath')
```

Comment:
I think we have an example of using this in the README. Are we sure we don't want to keep using the real file for consistency with the README or even just to make sure it actually works with formdata-node? Ideally another change should be included in this PR, either or both the README or the code, to cover both scenarios of files from the file system and blobs/files in memory

Reply:
Have reinstated the file from disk example, the jest environment needed to be updated to node for that case, so the tests are currently split into jsdom/node folders